components/observatorium.libsonnet

```diff
@@ -9,18 +9,23 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
       'app.kubernetes.io/part-of': 'observatorium',
       'app.kubernetes.io/instance': obs.config.name,
     },
+    replicaLabels:: ['prometheus_replica', 'rule_replica', 'replica'],
   },
 
   compact::
     t.compact +
     t.compact.withRetention +
-    t.compact.withDownsamplingDisabled + {
+    t.compact.withDownsamplingDisabled +
+    t.compact.withDeleteDelay +
+    t.compact.withDeduplication + {
       config+:: {
         local cfg = self,
         name: obs.config.name + '-' + cfg.commonLabels['app.kubernetes.io/name'],
         namespace: obs.config.namespace,
         replicas: 1,
         commonLabels+:: obs.config.commonLabels,
+        deduplicationReplicaLabels: obs.config.replicaLabels,
+        deleteDelay: '4h',
```

Comment:
Why 4h? No harm in default value, no?

Reply:
All right, makes sense. I will do the changes.